vuu/src/test/scala/org/finos/vuu/net/rpc/DefaultRpcHandlerTest.scala

```diff
@@ -0,0 +1,85 @@
+package org.finos.vuu.net.rpc
+
+import org.finos.vuu.net.{ClientSessionId, Error, JsonViewServerMessage, RequestContext, RpcCall, RpcResponse}
+import org.finos.vuu.viewport.{ViewPortRpcFailure, ViewPortRpcSuccess}
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class DefaultRpcHandlerTest extends AnyFlatSpec with Matchers with BeforeAndAfterEach {
+  var handler: DefaultRpcHandler = _
+  val ctx = RequestContext("requestId", ClientSessionId("sessionId", "user"), null, "token")
+
+  override def beforeEach(): Unit = {
+    handler = new DefaultRpcHandler
+  }
+
+  "The registerRpcMethodHandler method" should "register a handler for a given rpc method" in {
```

Comment:
All existing tests use AnyFeatureSpec. We don't have actual tests that use AnyFlatSpec.
Official guide mentions using AnyFlatSpec as default and  AnyFeatureSpec for more acceptance style
https://www.scalatest.org/user_guide/selecting_a_style

Let's have a quick decision as a team which style(s) we want to use + when so we can try and keep things more consistent

Reply:
This is not acceptance test TBH, valid point though on following already established pattern - rewritten tests as requested